docs/labs.md

```diff
@@ -196,3 +196,7 @@ Threading allows users to branch out a new conversation from the main timeline o
 Threads can be access by clicking their summary below the root event on the room timeline. Users can find a comprehensive list of threads by click the icon on the room header button.
 
 This feature might work in degraded mode if the homeserver a user is connected to does not advertise support for the unstable feature `org.matrix.msc3440`  when calling the `/versions` API endpoint.
+
+## Voice & video rooms (`feature_voice_rooms`)
```

Comment:
from the react-sdk side: should this be listed as under development?

Reply:
Yes it should